ibm/service/power/resource_ibm_pi_workspace.go

```diff
@@ -199,3 +235,17 @@ func isIBMPIResourceDeleteRefreshFunc(client *instance.IBMPIWorkspacesClient, id
 		}
 	}
 }
+
+func resourceIBMPIWorkspaceUpdate(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	if d.HasChange(Arg_UserTags) {
+		crn := d.Get(Attr_CRN)
+		if crn != "" {
```

Comment:
I believe you also need crn != nil since you are reading from d.get

Reply:
updated